config/config.exs

```diff
@@ -13,10 +13,11 @@ config :exchange_api,
 config :exchange_api, ExchangeApi.Repo,
   adapter: Ecto.Adapters.Postgres,
   username: System.get_env("POSTGRES_USER"),
-  password: System.get_env("POSTGRES_PASS")
-  database: System.get_env("POSTGRES_DB")
-  hostname: System.get_env("POSTGRES_HOST")
-  pool_size: System.get_env("POSTGRES_POOL_SIZE")
+  password: System.get_env("POSTGRES_PASS"),
+  database: System.get_env("POSTGRES_DB"),
+  hostname: System.get_env("POSTGRES_HOST"),
+  pool_size: System.get_env("POSTGRES_POOL_SIZE") |> String.to_integer()
+
```

Comment:
Maybe we can set some defaults so users can set environment variables if they want to, but everything still works without them